cdap-ui/app/cdap/components/Experiments/ExperimentsServiceControl/index.js

```diff
@@ -32,8 +32,18 @@ export default class ExperimentsServiceControl extends Component {
     onServiceStart: PropTypes.func
   };
 
+  componentDidMount() {
+    isSpark2Available()
+      .subscribe(
+        isAvailable => this.setState({
+          disabled: !isAvailable
+        })
+      );
```

Comment:
can there be a case where the `versions` API takes a while to return, and we are already showing the landing page?